languageserver/server/server.go

```diff
@@ -1730,6 +1732,10 @@ func (s *Server) InlayHint(
 
 	for _, variableDeclaration := range variableDeclarations {
 		targetType := checker.Elaboration.VariableDeclarationTargetTypes[variableDeclaration]
+		if targetType == nil { // bugfix getting nil targetType
```

Comment:
Skipping is an OK fix, but maybe add a TODO here, this should never occur

Reply:
Do you have a theory why this happens? You think it could be some sort of a race condition using checker? the checker might recheck during the execution of this function. Do you think we can submit stuff to Sentry so we debug this easier?

Reply:
I don't know off the top of my head, we'll need to look into it.